src/locales/de/ErrorPage.json

```diff
@@ -0,0 +1,4 @@
+{
+    "Subtitle": "Seite nicht gefunden",
+    "MainText": "Sie haben unsere Fehlerseite gefunden. Dies ist nicht Ihre Schuld, sondern ein Fehler auf unserer Serverseite. Bitte entschuldigen Sie die entstandenen Unannehmlichkeiten und kontaktieren Sie gern unser Team unter dem Reiter 'Probleme melden'."
+}
```

Comment:
I would rewrite this. A simple "Die Seite konnte nicht geladen werden" would be fine. We are not always responsible for network errors. And the user might also be responsible for the error. And the user is not the president of the united states who might get angry about "Unannehmlichkeiten"

Reply:
obsolete